project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
Yes :+1: , but the last thing I got from you is to have this component as a sub-project of Cromwell-engine and then during deployment pipeline it will create a two jar files (one for the engine an the other for the backend) and will push those artifacts into Broad Artifactory.
If you changed your mind, perfect! We can avoid changes in sbt file.
Please, let me know :).